modules/c/partials/release-notes/couchbase-mobile-c-vs-release-note.1.0.0-beta.adoc

```diff
@@ -1,3 +1,25 @@
+[#vs-maint-1-0-1]
+== 1.0.1 -- December 2025
```

Comment:
Maybe adding a parenthesis to indicate that it's android only?

Something like == 1.0.1 (Android Only) -- December 2025.